R/CAMPP2.R

```diff
@@ -107,24 +107,24 @@ runCampp2 <- function (data1, metadata1, data2=NULL, metadata2=NULL, technology,
     print("RUNNING MISSING VALUE IMPUTATIONS")
     print("Running missing values imputation on data1")
 
-    data1<-ReplaceNAs(data1)
+    SE_1<-ReplaceNAs(SE_1)
 
-    print("Missing values imputation on data1 has finished")
+    print("Missing values imputation on SE_1 has finished")
```

Comment:
does the user know what `SE_1` is at this point? If it's just our own internal representation we should keep calling them `dataX`, so the user knows what we are talking about, otherwise we should introduce what `SE_1` means

Reply:
I have added:
```
...
        print("Creating Summarized Experiment object (SE) from 1st dataset.")
...
        print("Creating SE from 1st dataset (SE_1) has finished.")

```